app/src/main/java/com/infomaniak/drive/ui/MainActivity.kt

```diff
@@ -202,7 +204,25 @@ class MainActivity : BaseActivity() {
         }
     }
 
+    // We use this SuppressLint because we don't want to execute performClick on profileItem when double tapping.
+    @SuppressLint("ClickableViewAccessibility")
     private fun setupBottomNavigation() = with(binding) {
+
+        val gestureDetector = GestureDetector(this@MainActivity, object : GestureDetector.SimpleOnGestureListener() {
+            override fun onDoubleTap(e: MotionEvent): Boolean {
+                mainViewModel.switchToNextUser { navController.navigate(R.id.homeFragment) }
+                return true
+            }
+
+            override fun onLongPress(e: MotionEvent) {
+                navController.navigate(R.id.switchUserActivity)
+            }
+        })
+
+        bottomNavigation.findViewById<View>(R.id.menuFragment).setOnTouchListener { _, event ->
```

Comment:
coulnd't this be find by binding ?

Reply:
I could not access a specific view of the BottomNavigation and set an onTouchListener, but maybe I'm missing something.